scripts/cpp_lint.py

```diff
@@ -52,6 +52,7 @@
 import sys
 import unicodedata
 
+PY_3K = (sys.version_info[0] == 3)
```

Comment:
Perhaps this should be:

`PY_3K = (sys.version_info[0] >= 3)`

It sounds like Python 4 will probably be mostly backwards compatible with Python 3 (Python 4 will be the version after Python 3.9).